src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -624,45 +624,45 @@ windowGridColorsReset = Grid Farbe auf den Logisim standart zurücksetzen
 #
 # prefs/HotkeyOptions.java
 #
-# ==> hotkeyOptMenuKeyHeader =
-# ==> hotkeyOptNormalKeyHeader =
-# ==> hotkeyOptOrientDesc =
+hotkeyOptMenuKeyHeader = Menü Shortcuts die %s beinhalten müssen:
+hotkeyOptNormalKeyHeader = Situationsabhängige Shortcuts die %s nicht benötigen:
+hotkeyOptOrientDesc = Componenten ausrichten
 # ==> hotkeyOptHelp =
-# ==> hotkeyOptTitle =
-# ==> hotkeyOptResetBtn =
-# ==> hotkeySimAutoPropagate =
-# ==> hotkeySimReset =
-# ==> hotkeySimStep =
-# ==> hotkeySimTickHalf =
-# ==> hotkeySimTickFull =
-# ==> hotkeySimTickEnabled =
-# ==> hotkeyEditUndo =
-# ==> hotkeyEditRedo =
-# ==> hotkeyWindowClose =
-# ==> hotkeyWindowMinimize =
-# ==> hotkeyFileExport =
-# ==> hotkeyFilePrint =
-# ==> hotkeyErrMeta =
-# ==> hotkeyErrConflict =
-# ==> hotkeyDirNorth =
-# ==> hotkeyDirSouth =
-# ==> hotkeyDirEast =
-# ==> hotkeyDirWest =
-# ==> hotkeyEditMenuDuplicate =
-# ==> hotkeyEditToolDuplicate =
-# ==> hotkeyProjMoveUp =
-# ==> hotkeyProjMoveDown =
-# ==> hotkeyAutoLabelOpen =
-# ==> hotkeyAutoLabelToggle =
-# ==> hotkeyAutoLabelView =
-# ==> hotkeyAutoLabelHide =
+hotkeyOptTitle = Shortcuts Einstellungen
+hotkeyOptResetBtn = Auf Standardeinstellungen zurücksetzen
+hotkeySimAutoPropagate = Automatisch weiterschalten
+hotkeySimReset = Simulation zurücksetzen
+hotkeySimStep = Simulation weiterschalten
+hotkeySimTickHalf = Einen halben Zyklus weiterschalten
+hotkeySimTickFull = Einen vollen Zyklus weiterschalten
+hotkeySimTickEnabled = Simulator kontinuierlich betreiben
+hotkeyEditUndo = Rückgängigmachen
+hotkeyEditRedo = Wiederherstellen
+hotkeyWindowClose = Fenster schließen
+hotkeyWindowMinimize = Fenster minimieren
+hotkeyFileExport = Datei exportieren
+hotkeyFilePrint = Design drucken
+hotkeyErrMeta = Der Shortcut muss die %s Taste enthalten. 
+hotkeyErrConflict = Überschneidung mit: %s.
+hotkeyDirNorth = Norden
+hotkeyDirSouth = Süden
+hotkeyDirEast = Osten
+hotkeyDirWest = Westen
+hotkeyEditMenuDuplicate = Duplizieren
+hotkeyEditToolDuplicate = Auswahl duplizieren
+hotkeyProjMoveUp = Schaltung nach oben
+hotkeyProjMoveDown = Schaltung nach unten
```

Comment:
I'm not happy with this translation because without the context it is difficult to guess what it does.

The English versions "Move the Circuit Up Layer" and "the Circuit Down Layer" are not that great either.

Reply:
The actual menu items in English are "Move Circuit Up" and "Move Circuit Down"

As I mentioned in #1861, the name in the panel should match the name in the menu. So this should be changed to "Move circuit up" (or "down") to match the case as done in #1861. The same concept should likely apply in the other languages. @MarcinOrlowski: could you change this in #1861? Or alternatively, would it work to actually take the string from the menu item and not duplicate the information?

Reply:
@MarcinOrlowski: I don't do translation to non-English languages, but I can appreciate why trying to automatically generate strings could cause problems.

Reply:
"Schaltung nach oben bewegen", "Schaltung nach unten bewegen"